content/docs/get-started/policy-as-code/_index.md

```diff
@@ -1,28 +1,26 @@
 ---
-title: Get Started with Policy as Code (Preview)
+title: Get Started with Policy as Code
 linktitle: Policy as Code
 menu:
   getstarted:
     identifier: pac
     weight: 3
 ---
-{{% pac-preview %}}
+{{% crossguard-preview %}}
```

Comment:
Not a big enough deal to change for this PR, but when dealing with the "internal codename" and "external, branding name" in my experience it is best to keep those two separate.

For example, keeping the shortcake as `pac-preview` and not renaming it to `crossguard-preview`. It makes things harder for people who work at Pulumi, and perhaps even external OSS contributors. But given the finicky nature of branding, naming, etc. it helps insulate us from a set of changes. (e.g. if we later call it "Pulumi Policy Pack" or "Cloud Secure" or "Policy Platform", or -- as Microsoft did ~2008 -- "Pulumi CrossGuard Live Policy as Code Server R2 Beta Express".